src/components/Redirect.js

```diff
@@ -0,0 +1,9 @@
+import { useCampaignConfig } from "@hooks/useConfig";
+
+const Redirect = () => {
+  const config = useCampaignConfig();
+  document.location = config.component.redirect.url;
```

Comment:
const url=config.component.redirect?.url; // so no error if the redirect isn't set
if (!url) alert ("ERROR: missing url to redirect to");